js/components/site-search.vue

```diff
@@ -63,11 +63,18 @@ export default {
         size: String,
         territoryId: String,
     },
+    created() {
+        const {query} = this.$location;
+
+        const originalField = this.$options.el.querySelector('#search');
+        const queryStringValue = decodeURIComponent(query.q.replace('+', ' '));
+
+        this.query = originalField.value || queryStringValue || '';
```

Comment:
This does the trick. We either pick:

1. the value at runtime of the search field (altered by user or provided by a `input[value]` attribute;
2. otherwise we pick it up from the query string;
3. otherwise field value has not been setup